history/taskana-simplehistory-rest-spring/src/test/java/pro/taskana/simplehistory/rest/TaskHistoryEventControllerIntTest.java

```diff
@@ -256,11 +264,34 @@ void should_GetSpecificTaskHistoryEventWithDetails_When_SingleEventIsQueried() {
                 "THI:000000000000000000000000000000000000"),
             HttpMethod.GET,
             restHelper.defaultRequest(),
-            ParameterizedTypeReference.forType(TaskHistoryEventRepresentationModel.class));
+            TASK_HISTORY_EVENT_REPRESENTATION_MODEL_TYPE);
 
     assertThat(response.getBody()).isNotNull();
     assertThat(response.getBody().getDetails()).isNotNull();
   }
 
+  @Test
+  void should_ThrowException_When_ProvidingInvalidFilterParams() {
```

Comment:
This is a duplicate. Please check `should_ReturnBadStatusErrorCode_When_InvalidQueryParameterIsUsed`